perf-tests/clusterloader2/pkg/framework/config/client_config.go

```diff
@@ -35,16 +38,62 @@ const (
 	burst       = 200
 )
 
+var muxInitializedMultiClientConfig sync.Mutex
+
+var extraAPIServerAddresses []string
+
+func SetAPIServers(apiServerAddresses string) error {
+	muxInitializedMultiClientConfig.Lock()
+	defer muxInitializedMultiClientConfig.Unlock()
+	if len(extraAPIServerAddresses) > 0 {
+		return fmt.Errorf("Api Server addresses already set [%v]", extraAPIServerAddresses)
+	}
+
+	addrs := strings.Split(apiServerAddresses, ";")
```

Comment:
need to check to each string is a valid string. 

This func may be helpful:
https://golang.org/pkg/net/#ParseIP

Reply:
The arg will have format as server from kubeconfig. It can be hostname or ip or dns, with or without port number. It will be too complicated to validate each scenario. Plus, the perf-test process will immediately exit if config is not correct and not be able to create a valid connection to api server. I don't think it is worthwhile to deal with argument validation here.